### ISSUE TITLE
AttributeError: 'Draw' object has no attribute 'tostring'

### ISSUE THREAD

**Issue by dipeshSam** [pytroll/aggdraw#99]
I want to draw on internal image memory, tried the given code in the [documentation](https://aggdraw.readthedocs.io/en/stable/), but following error has occurred. Also, I need this function very much.

```
Python 3.12.3 (tags/v3.12.3:f6650f9, Apr  9 2024, 14:05:25) [MSC v.1938 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import aggdraw
>>> d = aggdraw.Draw("RGB", (320, 200), "white")
>>> p = aggdraw.Pen("black", 0.5)
>>> d.line((0, 0, 500, 500), p)
>>> d.line((0, 500, 500, 0), p)
>>> s = d.tostring()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'Draw' object has no attribute 'tostring'
>>>
```
![Screenshot (197)](https://github.com/user-attachments/assets/936084a8-019f-4765-9526-92cdc3ce6861)
Any kind of help will be appreciated.
Thank you.

**Comment by djhoese** [comment]
Ah that was changed a long time ago when multiple aggdraw repositories were merged. You can see the pull request here:

https://github.com/pytroll/aggdraw/pull/12

The main thing was to match changes in Pillow (PIL) and Python itself where `tostring` was replaced by `tobytes` in Python 3 (if I'm remembering this correctly). It looks like all the actual code got updated, but not the example code.

So using `tobytes` should work. Let me know if it doesn't.

**Comment by dipeshSam** [comment]
Dear @djhoese,

I sincerely appreciate your kind reply and great support. While I was aware that the documentation might be old, I wasn’t aware of the crucial detail that `tostring` had been replaced by `tobytes`. After investigating the `tobytes` method, I can confirm that it functions correctly and perfectly fine.

_Screenshot attachment (solved):_
![successful_aggdraw_method](https://github.com/user-attachments/assets/1e52aa98-0db8-4956-9173-ff932cc9686c)
**Thank you immensely!** 😊
Regards

**Comment by djhoese** [comment]
Fixed in #100. Released as 1.3.19. Thanks for the bug report!
